addons/binding/org.openhab.binding.kostalinverter/src/main/java/org/openhab/binding/kostal/inverter/WebscrapeHandler.java

```diff
@@ -57,9 +62,9 @@ public void initialize() {
                 refresh();
                 updateStatus(ThingStatus.ONLINE);
             } catch (Exception e) {
+                logger.debug("Error refreshing source '{}'", getThing().getUID(), e);
                 updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
                         e.getClass().getName() + ":" + e.getMessage());
-                logger.debug("Error refreshing source {} ", getThing().getUID(), e);
             }
         }, 0, config.refreshInterval, TimeUnit.SECONDS);
```

Comment:
As suggested by @CWempe, this looks like a mismatch

Reply:
Done.